### ISSUE TITLE
Prevent event storm and race conditions

### ISSUE THREAD

**Issue by fvanroie** [HASwitchPlate/openHASP#119]
**Is your feature request related to a problem? Please describe.**
haswitchplate/openHASP-custom-component#36

**Describe the solution you'd like**
Reduce colorpicker value updates: update event handler like we did with sliders issue #88 and PR #111

**Describe alternatives you've considered**
n/a

**Additional context**
[video](https://user-images.githubusercontent.com/1550668/113125259-b6476480-9216-11eb-8b8e-2480bb51f9f2.mp4)

Colorpicker:
```
p2b6 => {"event":"down","color":"#8300ff","r":131,"g":0,"b":255}
p2b6 => {"event":"changed","color":"#7300ff","r":115,"g":0,"b":255}
p2b6 => {"event":"changed","color":"#6200ff","r":98,"g":0,"b":255}
p2b6 => {"event":"up","color":"#6200ff","r":98,"g":0,"b":255}
```

**Comment by nagyrobi** [comment]
Actually, we need this for all objects sending "changed". dropdown, slider, etc.

**Comment by fvanroie** [comment]
For normal objects and buttons, I've renamed "short" and "up" events to "up" and "release":
- `up` indicates a short click
- `release` indicates the plate is released after `long` or `hold`events

Both `up` or `release` states indicate the plate is no longer touched. While `down` is sent the object is touched.

**Comment by fvanroie** [comment]
Toggle, switch and checkbox have also been updated with an "up" and "down" `event` plus the `val` field.

**Comment by fvanroie** [comment]
This was implemented in 0.5.0
